### DISCUSSION TITLE
mixup performance

### DISCUSSION THREAD

**Discussion by xoofee** [ultralytics/yolov5#9815] [General]
I train yolov5s on a subset of coco with
  - case A:  2 classes: person and cars
  - case B: 10 classes: 
(1000 images per class and split 0.8/0.2 for train/val

I observe that
  in case A it converges fast when mixup set to 0.0 , compared to 0.1 mixup. Mixup decreate mAP a little (5%)
  in case B it converge in the same speed, how ever after 200 epochs, mAP with no mixup is better (21% v.s. 23%)

mAP@0.5 2 classes
![image](https://user-images.githubusercontent.com/7467604/196013547-4079c2a5-6140-4f53-af33-d017375a4f06.png)

mAP@0.5 10 classes
![image](https://user-images.githubusercontent.com/7467604/196019252-f8cb4b27-21aa-449a-83d0-ae3043133876.png)

the only difference is value of mixup

lr0: 0.1
lrf: 0.1
momentum: 0.937
weight_decay: 0.0005
warmup_epochs: 3.0
warmup_momentum: 0.8
warmup_bias_lr: 0.1
box: 0.05
cls: 0.3
cls_pw: 1.0
obj: 0.7
obj_pw: 1.0
iou_t: 0.2
anchor_t: 4.0
fl_gamma: 0.0
hsv_h: 0.015
hsv_s: 0.7
hsv_v: 0.4
degrees: 0.0
translate: 0.1
scale: 0.9
shear: 0.0
perspective: 0.0
flipud: 0.0
fliplr: 0.5
mosaic: 1.0
mixup: 0.0
copy_paste: 0.1

```
case A
                 Class     Images  Instances          P          R     mAP@.5 mAP@.5:.95:
                   all        352       2146      0.678      0.439      0.498       0.27
                person        352       1708      0.677      0.487      0.547      0.305
                   car        352        438      0.678       0.39      0.449      0.236

case B
             Class     Images  Instances          P          R     mAP@.5 mAP@.5:.95: 
               all        510       3923        0.5     0.0909     0.0666     0.0256
            person        510       1708       0.23      0.457      0.326      0.136
            bottle        510        438      0.179      0.263      0.151     0.0556
             chair        510        273          1          0     0.0103     0.0036
            banana        510        145          1          0    0.00203   0.000649
              book        510        272          1          0     0.0275    0.00955
               car        510        219          1     0.0115     0.0601     0.0231
           handbag        510        210          0          0    0.00137   0.000571
              boat        510        215      0.442     0.0465     0.0442     0.0145
              bird        510        234     0.0469     0.0214      0.015    0.00428
     traffic light        510        209      0.103       0.11     0.0286    0.00788
```

**Comment by glenn-jocher** [comment]
@xoofee mixup typically helps on COCO for very large models that tend to overfit earlier. If your model is not overfitting there is no reason to introduce augmentaiton.

**Reply by xoofee** [reply]
thanks jocher!
